appside/frames/body.js

```diff
@@ -215,15 +218,39 @@ class BodyMapFrame extends Frame {
             $(label).text(stmt);
             right.appendChild(label);
             right.appendChild(document.createElement('br'));
+
+            this.user_input.set(name, 'false'); // all unchecked
         }
 
+        // next will be implemented in navigator?
+        let next = document.createElement('button');
```

Comment:
Yes, Nav will create the Next button and this frame does not get to create its own Next button. Since BodyMapFrame does not control the Next button, it needs to update this.user_input data without the click of a button. One possible place to do this is inside of get_user_input.

Reply:
Fixed by putting operations into get_user_input()